packages/office-ui-fabric-react/src/components/Fabric/Fabric.base.tsx

```diff
@@ -29,7 +29,16 @@ export class FabricBase extends React.Component<
     });
     const divProps = getNativeProps<React.HTMLAttributes<HTMLDivElement>>(this.props, divProperties);
 
-    return <div {...divProps} className={classNames.root} ref={this._rootElement} />;
+    let fabricComponent;
+    // apply theme to just the div only if applyTheme is true, otherwise do not apply theme at all
+    fabricComponent = (
+      <div {...divProps} className={this.props.applyTheme ? classNames.rootThemed : classNames.root} ref={this._rootElement} />
```

Comment:
Shouldn't both be applied, rather than either/or?

Reply:
`classNames.root` doesn't have the backgroundColor for the theme so if applyTheme isn't true only that's appropriate but if it is, then we're using `classNames.rootThemed` which has  `color: root.textColor,
        backgroundColor: root.backgroundColor,` so that text & background color get the theme colors.

Reply:
This isn't really expected. In current patterns where style props control the styles, you'd pass in rootThemed as a style prop and conditionalize those sections of the styling.

In the world where we don't key styling off props, you'd apply both `root` and `rootThemed` classnames to the root. A developer might be applying styling to the `root` which you just dropped.

Reply:
Moved the conditionalizing into the styles file.